swap_meet/vendor.py

```diff
@@ -1,2 +1,62 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory if inventory is not None else []
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else:
```

Comment:
We don't need the `else` block, we can pull this into the main scope of the method.